### DISCUSSION TITLE
Request timeout  to get OID's

### DISCUSSION THREAD

**Discussion by rtrexmass** [akvorado/akvorado#927] [Q&A] [answered]
Hi, 

i'm currently running Akvorado under docker-compose on my Fedora PC, i'm unable to get flows from my Mikrotik router

The log in the inlet is this:

2023-10-26 17:41:44 {"level":"error","error":"request timeout (after 1 retries)","exporter":"172.20.0.1","time":"2023-10-26T20:41:44Z","caller":"akvorado/inlet/metadata/provider/snmp/poller.go:114","module":"akvorado/inlet/metadata/provider/snmp","message":"unable to GET (4 OIDs)"}

I have errors in the metrics API incrementing: 

akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="172.20.0.1"} 18429

By tcpdump watching, i am able to see that the packets are reaching my PC.

The ip that the "exporter" is using, is from docker internal network, could that be the problem? or docker can do the NAT without problems?, i have disabled userland_proxy as well, but the same error occurs.

**Comment by vincentbernat** [comment]
The exporter IP should be the real one. You can try to flush the conntrack with `conntrack -D -p udp`, but this should be handled automatically by the conntrack helper.

**Reply by rtrexmass** [reply]
Thanks for the reply,

I installed the conntrack-tool in my system, but the connections are cleared, and new ones appears as follows:
![image](https://github.com/akvorado/akvorado/assets/105231805/652563f0-8154-4b7d-ae67-bafcd2033008)
The error remains the same

**Comment by rtrexmass** [comment, marked as answer]
Ok i've found out my mistake, it's pretty stupid, but to someone struggling with this in the future, appears that when the docker engine is started by the docker desktop, the flags from the deamon.json are not passed, so it completed ignored it.

I've stard the docker with dockerd, then the containers and everything is working fine now, already seeing flows.
